src/utils.ts

```diff
@@ -825,6 +825,7 @@ const markdownEmojiTokenizer: marked.TokenizerExtension = {
 const markdownEmojiRenderer: marked.RendererExtension = {
   name: "emoji",
   renderer(token: marked.Tokens.Generic) {
+    console.log(token.emoji);
```

Comment:
please my console

Reply:
Done in [4cd702e](https://github.com/radicle-dev/radicle-interface/pull/371/commits/4cd702e412c295a7a2d65e26403f05cdc32d72d0)